src/wp-content/themes/twentytwelve/functions.php

```diff
@@ -217,8 +217,8 @@ function twentytwelve_scripts_styles() {
 	// Theme block stylesheet.
 	wp_enqueue_style( 'twentytwelve-block-style', get_template_directory_uri() . '/css/blocks.css', array( 'twentytwelve-style' ), '20240812' );
 
-	// Loads the Internet Explorer specific stylesheet.
-	wp_enqueue_style( 'twentytwelve-ie', get_template_directory_uri() . '/css/ie.css', array( 'twentytwelve-style' ), '20240722' );
+	// Register the Internet Explorer specific stylesheet.
+	wp_register_style( 'twentytwelve-ie', get_template_directory_uri() . '/css/ie.css', array( 'twentytwelve-style' ), '20240722' );
 	$wp_styles->add_data( 'twentytwelve-ie', 'conditional', 'lt IE 9' );
```

Comment:
I might take another step and remove the sources:
```suggestion
	wp_register_style( 'twentytwelve-ie', false, array( 'twentytwelve-style' ) );
```

T13, T15 and T17 would need the same, and I could replace the files' contents with a short comment (for all the themes).

Reply:
That makes sense to me.